_newsletters/2023-10-24-september-2023.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+category: newsletters
+title: September 2023 Newsletter
+filename-custom: september-2023
+last_modified_at: October 24, 2023
+sidenav: about_us
+newsletter-year: 2023
+description: "Government shutdown, search page alert, faceted search, UX panel discussion, review list of contacts, release notes."
+---
+
+## In case of a temporary lapse in appropriations
+
+### Use the search page alert to post a banner on your search results page
+
+In case of a temporary lapse of appropriations, or for any other reason, you can use our [Search Page Alert](https://search.gov/admin-center/display/system-alert.html) feature to post a banner on your search results page to inform searchers of your website’s status.
+
+### Search system will remain operational
+In the event of a temporary lapse in appropriations, please note that Search.gov will remain in normal operations and maintenance, including the search engine, hosted search results pages, and the Admin Center. API-driven search results pages on your websites are subject to your agency’s guidance.
+
+
+
+## Interested in faceted search?
+
+### Let’s talk about metadata!
+
+As previously announced, our faceted search capability is now available for our customers using API results, and it will soon become available to those customers using hosted results. If you are interested in using this feature for your website, email us at [search@gsa.gov](search@gsa.gov) to schedule a call to talk about getting your metadata ready. The sooner you can start including metadata in your page templates, the sooner we can gather it. We will start using it in results matching right away, and your searchers will be able to use it to filter when the interface is ready.
```

Comment:
the email link needs to be 
[search@gsa.gov](mailto:search@gsa.gov)

Can you update in both places, here and below, please?